DESCRIPTION

```diff
@@ -20,11 +20,15 @@ Description: The package contains manually curated data for the genomic Analysis
 License: MIT + file LICENSE
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.0
+RoxygenNote: 7.2.3
 Depends:
     R (>= 2.10)
 LazyData: true
 Imports: 
     dplyr,
     ggplot2,
-    tibble
+    tibble,
+    tidyr
+Remotes:
```

Comment:
Which helpers are used here? I think it also needs the GAMBLR.helpers to be listed in the `Imports` section